Tests-Legacy/L0/CodeAnalysis-Common/_suite.ts

```diff
@@ -0,0 +1,271 @@
+/// <reference path="../../definitions/mocha.d.ts"/>
```

Comment:
how come we're adding a legacy test?

Reply:
These tests were in the Maven legacy tests and don't need to be ported over.